src/inject/dynamic-theme/adopted-style-manger.ts

```diff
@@ -0,0 +1,93 @@
+import {Theme} from '../../definitions';
+import {createStyleSheetModifier} from './stylesheet-modifier';
+import {forEach} from '../../utils/array';
+
+let adoptedSheetOverride = new WeakMap<CSSStyleSheet, CSSStyleSheet>();
+let nodes = [] as Array<ShadowRoot | Document>;
+let overrides = new WeakSet<CSSStyleSheet>();
+
+export function removeAdoptedStyleSheets() {
```

Comment:
Now you have to do cleanup in `index.ts` file by calling `AdoptedStyleSheetManager.destroy()`